conf.toml

```diff
@@ -5,8 +5,9 @@ Type = "unix"
 
 [InstanceManager]
 Type = "unix"
-HostPort = 1080
-HostProtocol = "http"
+HostPort = 1443
```

Comment:
I think we need a new Setting class for these parameters, it could be [HostCommunication]. Having these parameters here is affecting the implementation bloating up the Instance Manager interface and I think the two concepts could be separated. We keep the IM responsibility for CRUD operations and a new component to deal with the communication challenges which is cloud provider agnostic.

Reply:
The way I see is is having the IM only responsible to provide the IP which is cloud provider specific, then we can have a new  common layer that is responsible for the http communication with the host orchestator, building the transport object, the http client and the reverse proxy. Putting it into the IM interface makes it bigger and I don't see a hard constrain for this.

Reply:
You're assuming that the Cloud Orchestrator only needs an IP address and port to communicate with the Host Orchestrator, I'm saying this won't always be the case.
If you want to have a class that specializes in CRUD and another one that specializes in comms, I'm ok with that, but those classes will be linked together at the implementation level, not the interface level. If you specify at the interface level the way you can reach the HO (by returning an IP and port) you'll be adding an unnecessary constraint.